mathics/builtin/box/graphics3d.py

```diff
@@ -63,40 +63,40 @@ def _prepare_elements(self, leaves, options, max_width=None):
 
         if lighting == "System`Automatic":
             self.lighting = [
-                {"type": "Ambient", "color": [0.3, 0.2, 0.4]},
+                {"type": "ambient", "color": [0.3, 0.2, 0.4]},
```

Comment:
Isn't this handled by the transformation layer in mathics-graphics3d-backend ?

Reply:
Yes, but is weird to have primitive names lower-case and light names capital-case.

Reply:
I wasn't saying this is wrong - just that mathics-threejs-backend _also_ needs to be able to handle things like this. 

The current plan is to release Mathics Django as is. And then _after_ that settles we will a slew of API breaking changes.

In general things, have breaking way too much recently. We need more control everywhere on limiting  breakage.

Reply:
Ok, so just remove the viewpoint scaling by now as it isn't a breaking change?

Reply:
With all of the changes it is hard to follow this PR.  It feels like after things settle down we should plain on a major Graphics3D in compact JSON design.  

Personally, I think that all of the design work and implementation in mathics-threejs-backend could go on now if you are so disposed. Nothing prevents creating whatever API  is best and converting what we have now to that.

In the PRs here in the short term we should separate breaking API from non-breaking. 

Adding fields to existing JSON is fine that isn't going to break older Mathics-Django that happen use Mathics inore dev. changing existing values would.  And having mathics-threejs-backend ignore existing fields is fine too.